content/docs/06-integrations/01-maintenance-policy.md

```diff
@@ -0,0 +1,181 @@
+import InfoBox from 'shared/components/InfoBox';
+
+
+
+# Overview
+
+Palette supports two pack categories: *Core* and *Add-on*. Core packs, which we often source from third parties, are infrastructure related or have prominence in container-based environments. They create the layers in Palette profiles. Core packs are grouped as follows:
+
+- Kubernetes
+
+- Operating System (OS)
+
+- Container Network Interface (CNI)
+
+- Container Storage Interface (CSI)
+
+
+Add-on packs provide additional functionality that you can add to your cluster profile and are grouped as follows:
+
+- Load Balancer
+
+- Ingress
+
+- Logging
+
+- Monitoring
+
+- Security
+
+- Authenticaiton
+
+- System Apps
+
+
+Check out the [Packs List](/integrations) document, where you can use the filter buttons to display a list of Palette packs in each category and learn about the individual packs.
+
+<br />
+
+## Pack Updates
+
+Packs undergo rigorous vulnerability scans and penetration testing before they become available in Palette. The following sections describe our update schedule for each core pack category. 
+
+<br />
+
+### Kubernetes Packs
+
+We provide Cloud Native Computing Foundation (CNCF) Kubernetes updates as follows:
+
+<br />
+
+- **Major versions**: Assessed based on the extent of changes.
+
+
+- **Minor versions**: Provided within eight weeks of a new Kubernetes release.
+
+
+- **Patch versions**: Provided within four weeks of a new Kubernetes release.
+
+
+
+### OS Packs
+
+We provide Ubuntu LTS and CentOS updates for IaaS clusters as follows:
+
+<br />
+
+- **Major versions** - Added within 8 weeks of release.
+
+
+- **Patch and Minor versions**:  Updated at runtime using Palette’s on-demand or scheduled OS upgrades and patch-on-boot capabilities.
+
+
+### CNI Packs
+
+We provide CNI pack updates as follows:
+
+<br />
+
+- **Major versions**: Assessed based on the extent of changes.
+
+
+- **Minor version**: Added within 6 weeks of release.
+
+
+- **Patch versions**: Added within 4 weeks of release.
+
+
+### CSI Packs
+
+We provide CSI pack updates as follows:
+
+<br />
+
+- **Major versions**: Assessed based on the extent of changes.
+
+
+- **Minor version**: Added within 6 weeks of release.
+
+
+- **Patch versions**: Added within 4 weeks of release.
+
+
+### Add-on Packs
+
+We provide add-on pack updates as follows:
+
+<br />
+
+- **Major versions**: Assessed based on the extent of changes.
+
+
+- **Minor version**: Added within 6 weeks of release.
+
+
+- **Patch versions**: Added within 4 weeks of release.
+
+
+## Packs Deprecation
+
+
+We deprecate and remove packs when a more stable version of the pack is available or when the underlying technology becomes obsolete.
+Below are the stages of deprecation we adhere to.
+
+<br />
+
+- **Deprecated**: When a pack is deprecated, Palette displays a warning. You will still be able to create new cluster profiles using the pack and launch clusters using existing profiles that contain the pack.
+
```

Comment:
Displaying a warning is coming up in upcoming release -4.0.0. For now, deprecated packs will be mentioned only in release notes.

Reply:
Revised to say the following for now and update it for 4.0. 

"Deprecated packs are documented in the Release Notes. You will still be able to create new cluster profiles using the pack and launch clusters using existing profiles that contain the pack."